platforms/quorum/charts/quorum-genesis/templates/genesis-job-init.yaml

```diff
@@ -34,12 +34,23 @@ spec:
           securityContext:
             runAsUser: 0
           imagePullPolicy: {{ .Values.image.pullPolicy }}
-{{- if eq .Values.global.vault.type "hashicorp" }}
+{{- if or (eq .Values.global.vault.type "hashicorp") (eq .Values.global.vault.type "aws-secrets-manager") }}
```

Comment:
condition is wrong, it should use the `global.cluster.cloudnativeservices`, not a separate one. because in this case cloud can be `azure` and then vault.type is `aws-secrets-manager` which would not work

Reply:
`global.cluster.cloudnativeservices` can be set to `aws`, and
`global.vault.type` value will remain as the default, which is `kubernetes`. Am I understanding it correctly?